Libplanet/Net/Swarm.cs

```diff
@@ -2150,12 +2150,24 @@ private async Task ProcessFillTxs(CancellationToken cancellationToken)
                         $"Some tasks faulted during {nameof(GetTxsAsync)}().");
                 }
 
-                foreach (var task in tasks.Where(task => !task.IsFaulted))
+                foreach (Task<List<Transaction<T>>> task in tasks.Where(task => !task.IsFaulted))
                 {
+                    // `task.Result` is okay because we've already waited.
                     txs.AddRange(task.Result);
                 }
 
-                BlockChain.StageTransactions(txs.ToImmutableHashSet());
+                foreach (Transaction<T> tx in txs)
+                {
+                    try
+                    {
+                        BlockChain.StageTransaction(tx);
+                    }
+                    catch (InvalidTxException ite)
+                    {
+                        _logger.Error(ite, "{tx} isn't valid; ignored.");
```

Comment:
```suggestion
                        _logger.Error(ite, "{tx} isn't valid; ignored.", tx);
```

```suggestion
                        _logger.Error(ite, $"{tx} isn't valid; ignored.");
```

Did you mean this? or *Serilog* provides some function?
Also `Transaciton<T>` doesn't provide `ToString()` method, so in my thought, it will show a message like `Transaction<ActionBase> isn't valid; ignored.`. How about logging `tx.Id` to help to specify the transaction?

Reply:
It's also simple mistake. 😅 . I've intended as seliog's template message.